### PR TITLE
Logic for setting correct/incorrect class on assessment components

### PR DESCRIPTION
**Pull request by mbareta**
This is completely optional so it won't make breaking changes.

### CODE REVIEW

**Comment by mbareta** [comment]
TODO: make this change optional to avoid breaking of other platforms.

File: src/teaching-element/Assessment/MultipleChoice.vue [hunk 1]
```diff
@@ -5,11 +5,11 @@
       <li
         v-for="(answer, index) in answers"
         :key="index"
-        :class="{ selected: isSelected(index) }">
+        :class="[{ selected: isSelected(index) }, getAnswerClass(index)]">
```
Comment (kronicker) [inline src/teaching-element/Assessment/MultipleChoice.vue]: Build the class array in `getAnswerClass ` method.

File: src/teaching-element/Assessment/MultipleChoice.vue [hunk 2]
```diff
@@ -45,16 +46,21 @@ export default {
     }
   },
   methods: {
-    update() {
-      let { userAnswer } = this;
-      userAnswer = isEmpty(userAnswer) ? null : userAnswer.sort();
-      this.$emit('update', { userAnswer });
+    getAnswerClass(answerIndex) {
+      if (!this.disabled || !this.options.showCorrectChoices) return null;
```
Comment (kronicker) [inline src/teaching-element/Assessment/MultipleChoice.vue]: Let's rename this `options` prop to `setCorrectnessClass` and also add it to all other assessment components so weather classes will be set can be controlled from the outside for any type of assessment. 
Also, let's default it to true.

File: src/teaching-element/Assessment/MultipleChoice.vue [hunk 3]
```diff
@@ -45,16 +46,21 @@ export default {
     }
   },
   methods: {
-    update() {
-      let { userAnswer } = this;
-      userAnswer = isEmpty(userAnswer) ? null : userAnswer.sort();
-      this.$emit('update', { userAnswer });
+    getAnswerClass(answerIndex) {
```
Comment (kronicker) [inline src/teaching-element/Assessment/MultipleChoice.vue]: I think `index` is sufficient.

**Pull request closed without merging**